catalog/spatial/wfs/spatial-wfs-metacardtype-registry/src/test/java/org/codice/ddf/spatial/ogc/wfs/catalog/metacardtype/registry/WfsMetacardTypeRegistryTest.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.catalog.metacardtype.registry;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
Should be using Hamcrest matchers : https://codice.atlassian.net/wiki/display/DDF/Unit+Testing